packages/core/src/functions.ts

```diff
@@ -40,21 +39,29 @@ export interface EvalValueResult {
 
 export class StylableEvaluator {
     public stVarOverride: Record<string, string> | null | undefined;
-    constructor(options: { stVarOverride?: Record<string, string> | null } = {}) {
+    public getResolvedSymbols: (meta: StylableMeta) => MetaResolvedSymbols;
+    public valueHook?: replaceValueHook;
+    constructor(options: {
+        stVarOverride?: Record<string, string> | null;
+        valueHook?: replaceValueHook;
+        getResolvedSymbols: (meta: StylableMeta) => MetaResolvedSymbols;
+    }) {
+        this.valueHook = options.valueHook;
         this.stVarOverride = options.stVarOverride;
+        this.getResolvedSymbols = options.getResolvedSymbols;
     }
     evaluateValue(
-        context: FeatureTransformContext,
-        data: Omit<EvalValueData, 'passedThrough'> & { passedThrough?: string[] }
+        context: Omit<FeatureTransformContext, 'getResolvedSymbols'>,
```

Comment:
you just added `passedThrough` to the transform context, and it also sounds like it belongs there more then to data.

Maybe pass is as part of the context and not the data?

Reply:
I added this in the last commit, and added a related issue I fount that wan't caused by any change in this PR: https://github.com/wix/stylable/issues/2820